cluster/common.sh

```diff
@@ -147,7 +150,8 @@ function clear-kubeconfig() {
 
   local kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
   # Unset the current-context before we delete it, as otherwise kubectl errors.
-  local cc=$("${kubectl}" config view -o jsonpath='{.current-context}')
+  cc=$("${kubectl}" config view -o jsonpath='{.current-context}')
+  local cc
```

Comment:
so this may be confusing, but for `export` it should come after assignment, while for `local` it should come _before_.

Reply:
thank you @BenTheElder , I will modify this.